src/dsl.ts

```diff
@@ -189,68 +191,89 @@ export function mapParsedSchema(
     return;
   }
 
-  schema.definitions.forEach((def: TopLevelDefinition) => {
-    mapParseNodes(def, mapper);
-  });
+  schema.definitions.forEach(mapParseNodes(mapper));
```

Comment:
I changed the signature of `mapParseNodes` so that it takes a function and returns a function, which makes these declarations a little nicer.